src/main/java/tempest/GUI/AddModulePage.java

```diff
@@ -0,0 +1,118 @@
+package tempest.GUI;
+
+import tempest.Module;
+import tempest.State;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+//TODO:
+// - Better if pressing enter sends user back a screen? - Feels more like you've entered something
+
```

Comment:
Would it be a good idea to move this out of the code to an issue, or something in the sprint diary?

Reply:
I've put something in the sprint diary